.bundle/config

```diff
@@ -0,0 +1,4 @@
+---
+BUNDLE_SPECIFIC_PLATFORM: "false"
+BUNDLE_PATH: "vendor/bundle"
+BUNDLE_JOBS: "16"
```

Comment:
This seems like an odd thing to hardcode into the project (what if I run `bundle` on a 4-core machine?)

Reply:
This is related to p91TBi-6it-p2